packages-content-model/roosterjs-content-model-api/lib/modelApi/list/setListType.ts

```diff
@@ -37,19 +37,16 @@ export function setListType(model: ContentModelDocument, listType: 'OL' | 'UL')
                 block.levels.push(level);
             } else if (block.blocks.length == 1) {
                 setParagraphNotImplicit(block.blocks[0]);
-                block.blocks.forEach(x => {
-                    if (block.format.marginLeft) {
-                        x.format.marginLeft = block.format.marginLeft;
-                    }
-
-                    if (block.format.marginRight) {
-                        x.format.marginRight = block.format.marginRight;
-                    }
-
-                    if (block.format.textAlign) {
-                        x.format.textAlign = block.format.textAlign;
-                    }
-                });
+                const listBlock = block.blocks[0];
```

Comment:
Move this line one line up. So that the original line 39 can also use this `listBlock`.

And add comment to explain why we need to copy these styles.